examples/HelloWorld/helloworld_graph.py

```diff
@@ -0,0 +1,45 @@
+"""
+This is an example of a very simple graph which prints hello for each even number x in the input stream,
+using a conditional RuleBasedModel node and a HelloPrinter h1.Action.
+"""
+
+import h1st as h1
+
+class RuleBasedModel(h1.Model):
```

Comment:
Is it possible to use the RuleBasedModel from `rule_based_model.py`? This is to help to see the re usability as well.

Reply:
We have to change RuleBasedModel from rule_based_model.py to echo back the _selected_ outputs, which I find odd because it should be H1st Graph who does the orchestration. Will create an issue to unify this dataflow.

Reply:
Yeah, the current implementation only filters the output of the decision node to pass down to yes/no flow. This is a new feature we can ask @thangtp to help with.

Reply:
I've updated the code to reuse the RuleBasedModel from rule_based_model.py.